src/brasil/gov/vlibrasnews/profiles/testfixture/types.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
```

Comment:
@rodfersou there is no need for this fake content type; use the ones on plone.app.contenttypes